direct/io-kafka/src/main/java/cz/o2/proxima/direct/kafka/KafkaAccessor.java

```diff
@@ -83,6 +90,8 @@
   @Getter(AccessLevel.PACKAGE)
   private int maxPollRecords = 500;
 
+  @Nullable Class<ElementSerializer<?, ?>> serializerClass;
```

Comment:
This should not be Nullable because of fallback to `KafkaStreamElementSerializer.class`